docs/content/docs/deployment/finegrained_resource.md

```diff
@@ -84,10 +84,12 @@ To use fine-grained resource management, you need to:
 
 ### Enable Fine-Grained Resource Management
 
-To enable fine-grained resource management, you need to configure the [cluster.fine-grained-resource-management.enabled]({{< ref "docs/deployment/config" >}}#cluster-fine-grained-resource-management-enabled) to true.
+The fine-grained resource management is enabled by default.
+
+To ensure this, you need to check that [cluster.fine-grained-resource-management.enabled]({{< ref "docs/deployment/config" >}}#cluster-fine-grained-resource-management-enabled) is not set to false.
 
 {{< hint danger >}}
-Without this configuration, the Flink runtime cannot schedule the slots with your specified resource requirement and the job will fail with an exception.
+When this configuration is disabled, the Flink runtime cannot schedule the slots with your specified resource requirement and the job will fail with an exception.
 {{< /hint >}}
```

Comment:
I think we can simply remove this section. Users now can use either coarse-grained or fine-grained resource management, by simply specifying fine-grained resource requirements in their (DataStream) codes or not, without having to worry about whether the feature is enabled.

Reply:
And the *Usage* section should also be updated, because "configure to enable fine-grained resource management" is no longer needed.